frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -3,9 +3,8 @@ import './Map.scss';
 import React, { useRef, useState, useEffect } from 'react';
 import axios from 'axios';
 import { LatLngBounds, LeafletMouseEvent } from 'leaflet';
-import { Map as LeafletMap, TileLayer, Marker, Popup } from 'react-leaflet';
-import { IParcel, IParcelDetail, IProperty, IPropertyDetail } from 'actions/parcelsActions';
-import { ParcelPopupView } from '../ParcelPopupView';
+import { Map as LeafletMap, TileLayer, Marker, Popup, WMSTileLayer } from 'react-leaflet';
+import { IProperty, IPropertyDetail } from 'actions/parcelsActions';
```

Comment:
Wonder if they should be renamed to `actions/propertyActions`?

Reply:
Agree, but given that most frontend code will be referencing these files I don't want to do it in this PR. I want to check with Devin whether he liked using `redux-toolkit` for defining our redux code (with less boilerplate and within a single file, instead of 3 files spread out in the solution)
If `redux-toolkit` is a go, I say we file a tech-debt item to refactor our state management code and **then** we would rename this as part of that work. What do you think?